src/index.html

```diff
@@ -7,6 +7,372 @@
+            <a class="nav__link" href="#samsung">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones-phones">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops-computers"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadjet">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photoes">
+              Фото
+            </a>
+          </li>
+          <li>
+            <a class="nav__link" href="#videos">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="content">
+      <div class="content__cards">
+        <div class="card content__card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="image of Apple monoblock"
+            class="card__image"
+          >
+          <div class="description card__description">
+            <div class="description__title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
+            <p>
+              Код товара: 195434
+            </p>
+          </div>
+          <div class="feedback card__feedback">
+            <div class="stars">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <p>
+              Отзывов: 5
+            </p>
+          </div>
+          <div class="price card__price">
+            <p>
+              Цена:
+            </p>
+            <div class="price__value">
+              69 999 грн
+            </div>
+          </div>
+          <a
+            href="#purchase"
+            class="link"
+            data-qa="card-hover"
+          >
+            Купить
+          </a>
+        </div>
+        <div class="card content__card">
+          <img
+            src="./images/imac.jpeg"
+            alt="image of Apple monoblock"
+            class="card__image"
+          >
+          <div class="description card__description">
+            <div class="description__title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
+            <p>
+              Код товара: 195434
+            </p>
+          </div>
+          <div class="feedback card__feedback">
+            <div class="stars">
```

Comment:
add modifier stars--4 and styles for all the modifiers, so that changing modifier in html would change number of active stars